Source/Core/Transforms.js

```diff
@@ -51,111 +53,187 @@ define([
+      up: [
+        0, 0, 1
+      ],
+      south: [
+        1, 0, 0
+      ],
+      west: [
+        0, -1, 0
+      ],
+      down: [0, 0, -1]
+    };
     /**
-     * Computes a 4x4 transformation matrix from a reference frame with an east-north-up axes
-     * centered at the provided origin to the provided ellipsoid's fixed reference frame.
-     * The local axes are defined as:
-     * <ul>
-     * <li>The <code>x</code> axis points in the local east direction.</li>
-     * <li>The <code>y</code> axis points in the local north direction.</li>
-     * <li>The <code>z</code> axis points in the direction of the ellipsoid surface normal which passes through the position.</li>
-     * </ul>
-     *
-     * @param {Cartesian3} origin The center point of the local reference frame.
-     * @param {Ellipsoid} [ellipsoid=Ellipsoid.WGS84] The ellipsoid whose fixed frame is used in the transformation.
-     * @param {Matrix4} [result] The object onto which to store the result.
-     * @returns {Matrix4} The modified result parameter or a new Matrix4 instance if none was provided.
-     *
-     * @example
-     * // Get the transform from local east-north-up at cartographic (0.0, 0.0) to Earth's fixed frame.
-     * var center = Cesium.Cartesian3.fromDegrees(0.0, 0.0);
-     * var transform = Cesium.Transforms.eastNorthUpToFixedFrame(center);
-     */
-    Transforms.eastNorthUpToFixedFrame = function(origin, ellipsoid, result) {
+    * Generates a function that computes a 4x4 transformation matrix from a reference frame
+    * centered at the provided origin to the provided ellipsoid's fixed reference frame.
+    * @param  {String} firstAxis  name of the first axis of the local reference frame. Must be
+    *  'east', 'north', 'up', 'west', 'south' or 'down'.
+    * @param  {String} secondAxis  name of the second axis of the local reference frame. Must be
+    *  'east', 'north', 'up', 'west', 'south' or 'down'.
+    * @return {localFrameToFixedFrameGenerator~resultat} The function that will computes a
+    * 4x4 transformation matrix from a reference frame, with first axis and second axis compliant with the parameters,
+    */
+    Transforms.localFrameToFixedFrameGenerator = function( firstAxis, secondAxis) {
+      if (!vectorProductLocalFrame.hasOwnProperty(firstAxis) || !vectorProductLocalFrame[firstAxis].hasOwnProperty(secondAxis)) {
+        throw new DeveloperError('firstAxis and secondAxis must be east, north, up, west, south or down.');
+      }
+      var thirdAxis = vectorProductLocalFrame[firstAxis][secondAxis];
+      var calculateCartesian = {
+        east: new Cartesian3(),
+        north: new Cartesian3(),
+        up: new Cartesian3(),
+        west: new Cartesian3(),
+        south: new Cartesian3(),
+        down: new Cartesian3()
+      };
+      var firstCartesian = new Cartesian3();
+      var secondCartesian = new Cartesian3();
+      var thirdCartesian = new Cartesian3();
+
+      /**
+       * [resultat description]
```

Comment:
You can create function doc for the returned function. See [here](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Billboard.js#L1290) for an example.